### PR TITLE
Add Makefile and update README.md for that

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
 ```
 
-### Launch the Operator
-```bash
-# Deploy the app-operator
-$ kubectl create -f deploy/rbac.yaml
-$ kubectl create -f deploy/crd.yaml
-$ kubectl create -f deploy/operator.yaml
+### Build it
+`make build`
```
Comment (irosenzw) [inline README.md]: Maybe and '$' and wrap it like that : 
```bash
$ make build
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
 
-# Create the operator resource (kubectl get apps)
-$ kubectl create -f deploy/cr.yaml
-```
+### Generate the Operator Container with the operator-sdk
+`make generate`
```
Comment (irosenzw) [inline README.md]: same here

File: README.md [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
+
+### Launch the Operator
+`make deploy`
```
Comment (irosenzw) [inline README.md]: same here

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+TAG?=$(shell git rev-parse --short HEAD)
+
+GOLANG_FILES:=$(shell find . -name \*.go -print)
+pkgs = $(shell go list ./... | grep -v /vendor/ )
+
+.PHONY: dep
+dep:
+	dep ensure -v
+
+.PHONY: all
```
Comment (rthallisey) [inline Makefile]: You can combine all the `.PHONY:` sections into a single `.PHONY` at the bottom.

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+clean:
+	# Remove all files and directories ignored by git.
+	git clean -Xfd .
+
+############
+# Building #
+############
+
+.PHONY: build
+build: kubevirt-operator
```
Comment (rthallisey) [inline Makefile]: I would call this compile

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+kubevirt-operator: $(GOLANG_FILES)
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build \
+	-o $@ cmd/kubevirt-operator/main.go
+
+##############
+# Generating #
+##############
+
+.PHONY: generate
```
Comment (rthallisey) [inline Makefile]: I think calling this `build` is better

File: README.md [hunk 7]
```diff
@@ -1,18 +1,33 @@
...
+- operator-sdk
+
+### Dependencies
+
+Checkout the project and its dependencies:
+
 ```bash
-operator-sdk build docker.io/rthallisey/kubevirt-operator
+$ mkdir -p $GOPATH/src/github.com/rthallisey
+$ cd $GOPATH/src/github.com/rthallisey
```
Comment (ohadlevy) [inline README.md]: if you want folks to copy-paste this, maybe drop the ```$```

File: README.md [hunk 8]
```diff
@@ -1,18 +1,33 @@
...
 
-### Build the Operator Container
+## Quick Start
+
+### Prerequisites
+
+- golang environment and GOPATH correctly set
+- docker (used for creating container images, etc.) with access for the current user
+- a kubernetes/openshift/minikube/minishift instance
+- operator-sdk
```
Comment (alexxa) [inline README.md]: @tiraboschi as you capitalize 'Operator', it also will be good to use
* Golang
* Docker
* Kubernetes/OpenShift/Minikube/Minishift
* Operator SDK

**Pull request closed without merging**